equalitie/ceno
# Error reports

I'll write up API docs but please test.

To test:

1. Get list of RSS feeds from me
2. in `ceno/ceno-reader`, run `./build.sh`
3. Run the compiled `reader`, `bundleinserter`, and `bundleserver` executables
4. In `ceno/ceno-reader/tools`, run `go build follower.go`
5. Run `./follower sources.list` (retrieved from me)
6. After following is done, run `curl http://localhost:3096/errors`. You should see some error reports.
7. Run `curl http://localhost:3096/errors` again. You should see either no or few error reports.
